docs/source/commands/persisted-queries.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: The Rover persisted-queries command
+description: Publish persisted query manifests to your persisted queries list
+minVersion: 0.17.2
+---
+
+import AuthNotice from '../../shared/auth-notice.mdx';
+
+With [GraphOS Enterprise](/graphos/enterprise/), you can enhance a graph's security by maintaining a **persisted query list** (**PQL**) for your graph's self-hosted router. The [Apollo Router](/router/) checks incoming requests against the PQL, a safelist of trusted operations made by your first-party apps.
+
+Preregistering trusted operations to a PQL has two steps:
+
+1. Generating persisted queries manifests (PQM) using client-specific tooling
+2. Publishing PQMs to the PQL using Rover
+
+This article details the latter. For more information on PQM generation and other implementation steps, see the [GraphOS persisted queries documentation](/graphos/operations/persisted-queries).
+
+## Publishing a PQM
+
+### `persisted-queries publish`
+
+<AuthNotice />
+
+You can use Rover to publish a PQM for any client to any [existing PQL](/graphos/operations/persisted-queries#1-pql-creation-and-linking). 
+
+Run the `persisted-queries publish` command, like so:
+
+```bash
+rover persisted-queries publish my-graph@my-variant \
+  --manifest ./persisted-query-manifest.json
+```
+
+- The `my-graph@my-variant` argument is the [`graph ref`](../conventions#graph-refs) of _any_ variant the PQL is [linked to](/graphos/operations/persisted-queries#12-link-the-pql-to-variants).
+    - Graph refs have the format `graph-id@variant-name`.
+- Use the `--manifest` option to provide the path to the manifest you want to publish.
+
+Instead of `my-graph@my-variant`, you can also use the `--graph-id` and `--list-id` options to specify the PQL you want to publish to:
```

Comment:
```suggestion
Instead of `my-graph@my-variant`, you can combine the `--graph-id` and `--list-id` options to specify the PQL you want to publish to:
```

i think using `combine` makes it clearer that both arguments are required in this command form